### PR TITLE
[TIC-41] UI Fixes in Admin

### PR DESCRIPTION
**Pull request by henrykaus**
### Description
* Increased the padding bottom of `<h1>` elements in admin pages while reducing the margin to not cut off descending letters.
* Fixed typos.
* Added gathering of auth token for user.
* Fixed eslint warnings appearing in Docker
* Added margin to bottom of events selection screen in admin

#### Before
image

#### After
image

### Risks
N/A

### Validation
Visually tested locally.

### Issue
Resolves: #251 among other UI issues.

### Operating System
Windows 11

### CODE REVIEW

**Review by HaliTran — DISMISSED** [review]
LGTM!

File: client/src/components/CRM/Accounts/ManageUsers/ManageAccounts.tsx [hunk 1]
```diff
@@ -88,12 +88,21 @@ export default function ManageAccounts() {
   };
   const submitNewUser = async (e: any) => {
     e.preventDefault();
+    /** @param {string} token - AccessToken */
+    const token = await getAccessTokenSilently({
+      audience: 'https://localhost:8000',
+      scope: 'admin',
+    });
     /** @param {Response} r - Post data to Accounts Table */
     const r = await fetch(process.env.REACT_APP_API_1_URL + `/accounts`, {
```
Comment (HaliTran) [inline client/src/components/CRM/Accounts/ManageUsers/ManageAccounts.tsx:97]: This was probably already part of the code but I don't think using JSDoc for API calls is necessary. Renaming the const r variable to be more descriptive to what the data returns represents is self-documenting enough.

**Review by HaliTran — COMMENTED** [review]
After pulling and reviewing the changes, it looks good on my end as well. Code looks a lot more structured with the formatting changes. Left a single comment regarding JSdoc for API calls throughout the file.

File: client/src/components/CRM/Accounts/ManageUsers/ManageAccounts.tsx (hunk 1 above)
Comment (henrykaus) [reply client/src/components/CRM/Accounts/ManageUsers/ManageAccounts.tsx]: I removed the JSDoc comments for these cases, renamed the variables and also reformatted the JSX while I was at it

**Review by HaliTran — APPROVED** [review]

**Pull request merged**